projects/admin/src/app/record/detail-view/budget-detail-view/budget-detail-view.component.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Component, OnInit } from '@angular/core';
+import { DetailRecord } from '@rero/ng-core/lib/record/detail/view/detail-record';
+import { Observable } from 'rxjs';
+import { TranslateService } from '@ngx-translate/core';
+import { BudgetTotalService } from '../../../service/budget-total.service';
+import { OrganisationService } from '../../../service/organisation.service';
+
+@Component({
+  selector: 'admin-budget-detail-view',
+  templateUrl: './budget-detail-view.component.html'
+})
+export class BudgetDetailViewComponent implements DetailRecord, OnInit {
+
+  /**
+   * Record observable
+   */
+  record$: Observable<any>;
+
+  /**
+   * Type
+   */
+  type: string;
```

Comment:
useless?

Reply:
If you implement DetailRecord, this parameter is mandatory.